RELEASE.md

```diff
@@ -1,408 +1,290 @@
+  * Serveral changes have been made to `tf.keras.mixed_precision.experimental`. Note that it is now recommended to use the non-experimental `tf.keras.mixed_precision` API.
+    * `AutoCastVariable.dtype` now refers to the actual variable dtype, not the
+       dtype it will be casted to.
+    * When mixed precision is enabled, `tf.keras.layers.Embedding` now outputs a
+      float16 or bfloat16 tensor instead of a float32 tensor.
+    * The property
+      `tf.keras.mixed_precision.experimental.LossScaleOptimizer.loss_scale` is 
+      now a tensor, not a `LossScale` object. This means to get a loss scale of
+      a `LossScaleOptimizer` as a tensor, you must now call `opt.loss_scale`
+      instead of `opt.loss_scale()`.
+    * The property `should_cast_variables` has been removed from
+      `tf.keras.mixed_precision.experimental.Policy`
+    * When passing a `tf.mixed_precision.experimental.DynamicLossScale` to
+      `tf.keras.mixed_precision.experimental.LossScaleOptimizer`, the
+      `DynamicLossScale`'s multiplier must be 2.
+    * When passing a `tf.mixed_precision.experimental.DynamicLossScale` to
+      `tf.keras.mixed_precision.experimental.LossScaleOptimizer`, the weights of
+      the `DynanmicLossScale` are copied into the `LossScaleOptimizer` instead
+      of being reused. This means modifying the weights of the
+      `DynamicLossScale` will no longer affect the weights of the
+      LossScaleOptimizer, and vice versa.
+    * The global policy can no longer be set to a non-floating point policy in
+      `tf.keras.mixed_precision.experimental.set_policy`
+    * In `Layer.call`, `AutoCastVariable`s will no longer be casted within
+      `MirroredStrategy.run` or `ReplicaContext.merge_call`. This is because a
+      thread local variable is used to determine whether `AutoCastVariable`s are
+      casted, and those two functions run with a different thread. Note this
+      only applies if one of these two functions is called within `Layer.call`;
+      if one of those two functions calls `Layer.call`, `AutoCastVariable`s will
+      still be casted.
 
-    *   `tf.nn.max_pool2d` now supports explicit padding.
+* `tf.data`:
+  * `tf.data.experimental.service.DispatchServer` now takes a config tuple instead of individual arguments. Usages should be updated to      `tf.data.experimental.service.DispatchServer(dispatcher_config)`.
+  * `tf.data.experimental.service.WorkerServer` now takes a config tuple instead of individual arguments. Usages should be updated to  `tf.data.experimental.service.WorkerServer(worker_config)`.
+  
+* `tf.distribute`:
+  * Removes `tf.distribute.Strategy.experimental_make_numpy_dataset`. Please use `tf.data.Dataset.from_tensor_slices` instead.
+  * Renames `experimental_hints` in `tf.distribute.StrategyExtended.reduce_to`, `tf.distribute.StrategyExtended.batch_reduce_to`,      `tf.distribute.ReplicaContext.all_reduce` to `options`:
```

Comment:
there is a `:` at the end. was the intention to show code example there?

Reply:
Remved